react-native-libraries.json

```diff
@@ -13190,5 +13190,15 @@
     "npmPkg": "react-native-in-app-updates",
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/MihirGrand/react-native-serialport-windows",
+    "npmPkg": "react-native-serialport-windows",
+    "examples": [
+      "https://github.com/MihirGrand/react-native-serialport-windows/tree/main/example"
+    ],
+    "images": ["https://github.com/user-attachments/assets/f5349528-211a-4d91-88d6-725ac754725f"],
```

Comment:
Hey @MihirGrand, thanks for adding new entry to the directory! 👍 

CI fails because you have used GitHub camo link which requires token fetch to be displayed. A solution for that would be to upload the image differently to GH, upload it to some other hosting service, or just remove the asset from entry.